app/electron/main.ts

```diff
@@ -348,6 +380,46 @@ function getDefaultAppMenu(): AppMenu[] {
         },
       ],
     },
+    {
+      label: i18n.t('Navigate'),
+      id: 'original-navigate',
+      submenu: [
+        {
+          label: i18n.t('Reload'),
+          role: 'forcereload',
+          id: 'original-force-reload',
+        },
+        sep,
+        {
+          label: i18n.t('Go to Home'),
+          role: 'homescreen',
+          id: 'original-home-screen',
+          click: () => {
+            mainWindow?.loadURL(startUrl);
+          },
+        },
+        {
+          label: i18n.t('Go Back'),
+          role: 'back',
+          id: 'original-back',
+          accelerator: getAcceleratorForPlatform('Alt+Left', 'Cmd+Left', 'Alt+Left'),
```

Comment:
I'd prefer to call this function with only one param: Left | Right. Then inside the function should decide what to output.
Also, did you verify if Option+Left works on Mac? See:
https://www.electronjs.org/docs/latest/api/accelerator/#platform-notice
https://support.apple.com/guide/safari/keyboard-and-other-shortcuts-cpsh003/mac (it says it's `Command+[`)

Reply:
updated, thanks @joaquimrocha